tests/runtime/conftest.py

```diff
@@ -323,3 +376,21 @@ def _check_server_readiness(server):
         except grpc.RpcError:
             log.debug("[RpcError]; will try to reconnect to test server in 0.1 second.")
             time.sleep(0.1)
+
+
+def _check_http_server_readiness(server):
+    """Check server readiness"""
+    done = False
+    while not done:
+        try:
+            response = requests.get(
+                f"http://localhost:{server.port}{http_server.HEALTH_ENDPOINT}"
+            )
+            assert response.status_code == 200
+            assert response.text == "OK"
+            done = True
+        except AssertionError:
+            log.debug(
+                "[HTTP server not ready]; will try to reconnect to test server in 0.1 second."
+            )
+            time.sleep(0.1)
```

Comment:
Channeling Joe, but let's make this `0.01` so that it those tenths of a second don't add up!